### ISSUE TITLE
Calls to Office.context.mailbox.item.body.getAsync cause intermittent failures

### ISSUE THREAD

**Issue by feannyn** [OfficeDev/office-js#3189]
> Message from office-js bot:
We’re closing this issue because it has been inactive for a long time. We’re doing this to keep the issues list manageable and useful for everyone. If this issue is still relevant for you, please create a new issue.
> Thank you for your understanding and continued feedback.

We make a promisified call to `Office.context.mailbox.item.body.getAsync` when our Add-in in Outlook is pinned. When rapidly changing between mailbox items with large message bodies and a few attachments, there is an intermittent failure where the message body is not retrieved successfully. However, on a subsequent refresh of our add-in, the message body is retrieved successfully. Typically, we see this after toggling between mailbox items anywhere from 6 to 20 times with our add-in opened and pinned.

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: OWA
* Host [Excel, Word, PowerPoint, etc.]: Outlook
* Office version number: 
* Operating System: Ventura 13.2.1
* Browser (if using Office on the web): Google Chrome
 
## Expected behavior
When making a call to `Office.context.mailbox.item.body.getAsync`, we should receive the body of the specified item without issue, regardless of however many times we switch between  mailbox items.

## Current behavior
Currently, if the user is switching between mailbox items, there is an intermittent issue where the message body is not returned and it causes the add-in to crash; However, a refresh of the add-in will resolve this issue. The user will see this when rapidly switching between mailbox items with a few attachments and a large message body after the add-in has been pinned. 

## Steps to reproduce

1. Compose new message in OWA
2. The add-in needs to be side loaded, this is the link to the manifest (https://cdn.virtru.com/outlookjs/manifest/VirtruForOutlook.xml)
3. Open the Virtru Add-in
4. Enable the Virtru add-in, if not activated, activate the add-in
5. Compose a email with a large message body and a few attachments
6. Send the email
7. Do steps 1-5 so that you have a couple messages in the sent folder with a large message body and attachments
8. Once step 6 is complete, open one of the sent messages from step 6
9. Open the Virtru add-in
10. Pin the add-in
11. Begin switching between mailbox items at a rapid pace.

## Context
This issue is intermittent, If the add-in is refreshed, the user can continue as normal, but we have seen this with enough frequency to warrant the creation of this bug ticket.

## Useful logs
<img width="971" alt="Screenshot 2023-02-27 at 2 03 07 PM" src="https://user-images.githubusercontent.com/17707848/221658316-13fecfc8-95be-4d2d-90b9-502a74bf8641.png">

```
const getItemBody = (type = 'html') => new Promise((resolve, reject) => {
    Office.context.mailbox.item.body.getAsync(type, bodyTextAsyncResult => {
        if (bodyTextAsyncResult.status !== Office.AsyncResultStatus.Succeeded) {
            console.error(`Error getting item body: <${type}>`, bodyTextAsyncResult.error);
            reject(new Error(`Could not retrieve message body <${type}>`));
            return;
        }

        resolve(bodyTextAsyncResult.value);
    });
});
```

Thank you for taking the time to report an issue. Our triage team will res
pond to you in less than 72 hours. Normally, response time is <10 hours Monday through Friday. We do not triage on weekends.

**Comment by alison-mk** [comment]
Hi @feannyn, thank you for submitting this issue! I've assigned it to the engineering team who will be able to provide you with more information or help resolve the issue. 

Cheers,
Alison

**Comment by exextoc** [comment]
Thanks for reporting this issue. It has been put on our backlog. We unfortunately have no timelines to share at this point
Internal tracking id: 196030
